tests/endtoend/test_eventhub_functions.py

```diff
@@ -96,4 +96,5 @@ def test_eventhub_trigger_with_metadata(self):
         self.assertIsNotNone(sys_props['Offset'])
 
         # Metadata check eventhub trigger name
-        self.assertEqual(metadata['sys']['MethodName'], 'metadata_trigger')
+        # V3 Functions Host no longer pass 'sys' property to python sdk
+        # self.assertEqual(metadata['sys']['MethodName'], 'metadata_trigger')
```

Comment:
Same here?
Also, does this need to be documented for python's documentation - the change in the metadata?

Reply:
No, this is not documented. The customer will access the metadata field as a dictionary.